### DISCUSSION TITLE
HX Stomp Sidekick

### DISCUSSION THREAD

**Discussion by mattzzw** [alf45tar/PedalinoMini#438] [Show and tell]
This is a little midi unit I built for my HX Stomp based pedalboard. I just needed a couple of additional buttons to control effects. 

While I played around with double tapping in order to switch to snapshot or scroll-preset mode this quickly became too fiddly for me especially in a gigging situation. As I had a little real estate left on my pedal board I just added a simple dual switch for changing presets.

So I just use the Pedalino Sidekick mostly to send fixed CC messages per button and toggle the tuner via long press. 

![-5469712122599556340_121](https://user-images.githubusercontent.com/1312120/224659984-124092b5-49f4-4abf-a4c2-acec75f85838.jpg)
![-5444935091924092317_121](https://user-images.githubusercontent.com/1312120/224660019-b9855af8-1314-4d97-8c4a-e934f58e00b2.jpg)
![-5444935091924092316_121](https://user-images.githubusercontent.com/1312120/224660050-275f4a8d-38e4-4cfd-a68f-897350ae4421.jpg)

**Comment by zavorra** [comment]
That's great,  and using FreeCAD is a bonus!
would you mind to share the  design?

**Comment by mattzzw** [comment]
Here is the design, have fun! (And  please send pics if you used it or changed it)

[Midi_Muppet_v4.zip](https://github.com/alf45tar/PedalinoMini/files/11732605/Midi_Muppet_v4.zip)

**Comment by zavorra** [comment]
Thank you!

Il mar 13 giu 2023, 10:40 mattzzw ***@***.***> ha scritto:
